packages/php-wasm/web/vite.config.ts

```diff
@@ -57,7 +57,7 @@ export default defineConfig(({ command }) => {
 						 * and not
 						 * import("php_8_2.js")
 						 */
-						return '../' + specifier.split('/').pop();
+						return '../' + specifier.split('/').slice(-2).join('/');
```

Comment:
What's the reason for this change?

Reply:
This will correct the paths inside `@php-wasm/web`'s index.js file:

https://www.npmjs.com/package/@php-wasm/web?activeTab=code
index.js
scroll to line 1490

Reply:
Done. https://github.com/WordPress/wordpress-playground/pull/773/files#diff-b9f9449aad40f58f9a350e7e4a0ed52a2ca2480e197b2b587e9d5b581a40d7a0R60